extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -519,4 +519,13 @@ private boolean isEqual(State state1, State state2) {
 
         return configStatusMessages;
     }
+
+    @Override
+    public String getDeviceId() {
+        try {
+            return bridge.getFullConfig().getConfig().getMACAddress().replaceAll(":", "").toLowerCase();
+        } catch (Exception e) {
```

Comment:
I feared you are just covering a `catch (NPE)` here. Veto too!
Use the value from the handler configuration and your problem will vanish.